pkg/apis/batch/types.go

```diff
@@ -44,13 +44,23 @@ const (
 	JobNameLabel = labelPrefix + LegacyJobNameLabel
 	// Controller UID is used for selectors and labels for jobs
 	ControllerUidLabel = labelPrefix + LegacyControllerUidLabel
+	// Label indicating the controller that manages a Job. Jobs with a custom value
+	// of this label - any label value other than `job-controller.k8s.io` - are
+	// skipped by the job controller, and their reconciliation is delegated to
+	// an external controller, indicated by the value of the label. Jobs that
+	// don't have this label at all or where the label value is the reserved
```

Comment:
Active voice here too.

Reply:
Done, I have also rephrased a little to make it shorter. PTAL.